edge/services/helloMMS/Makefile

```diff
@@ -81,6 +81,11 @@ publish-only:
 	ARCH=arm64 $(MAKE) publish-service-overwrite
 	hzn exchange pattern publish -f horizon/pattern-all-arches.json
 
+publish-only-pattern-each-arch:
```

Comment:
We talked about adding `$(MAKE) publish-only-pattern-each-arch` to `publish-only` to invoke this new target

Reply:
done